msteams-platform/toolkit/teams-toolkit-migration-guide.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Migrate to new project structure
+author: v-preethah
+description: This guide provides the steps to migrate from scaffolded template to the new project structure in Microsoft Teams using Teams Toolkit.
+ms.author: v-preethah
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 06/05/2024
+---
+
+# Migrate to Microsoft Teams new project structure
+
+This guide details the process of migrating from the traditional, scaffolded template to the new project structure in Microsoft Teams. The transition is essential to leverage the latest features and improve project maintainability. The new project structure introduces a more streamlined and efficient way of managing your Teams applications.
+
+Learn in detail to create a new project folder, move configuration files, adjust launch settings, and more.
+
+## Prerequisites
+
+Before starting the migration process, ensure that you have the following:
+
+* Visual Studio 17.10 Preview 3 or later.
```

Comment:
A Teams project that is created using Teams Toolkit for Visual Studio 17.9 or earlier version.

Reply:
Updated